modules/nw-ingress-setting-thread-count.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-setting-thread-count_{context}"]
+= Setting Ingress Controller thread count
+A cluster administrator can set the thread count to increase the amount of incoming connections a cluster can handle. You can patch an existing Ingress Controller to increase the amount of threads.
+
+. Procedure
+The following assumes that you already created an Ingress Controller.
+
+.. Update the Ingress Controller to increase the number of threads
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontroller/default --type=merge -p '{"spec":{"tuningOptions": {"threadCount": 8}}}'
+----
+
+[IMPORTANT]
+====
+If you have a node that is capable of running large amounts of resrouces, you can configure `spec.nodePlacement.nodeSelector` with labels that match the capacity of the intended node, and configure `spec.threading.count` to an appropriately high value.
```

Comment:
Actually, spotted one change: `spec.threading.count` should be `spec.tuningOptions.threadCount` here